erpnext/public/js/utils/party.js

```diff
@@ -224,6 +224,10 @@ erpnext.utils.set_taxes = function(frm, triggered_from_field) {
 		party = frm.doc.party_name;
 	}
 
+	if (!frm.doc.company) {
+		frappe.throw(_("Kindly select the company first"));
```

Comment:
It's JS. You need to use double underscore for translation.
```suggestion
		frappe.throw(__("Kindly select the company first"));
```